core-blocks/video/index.js

```diff
@@ -75,16 +82,17 @@ export const settings = {
 	edit,
 
 	save( { attributes } ) {
-		const { autoplay, caption, controls, loop, muted, src } = attributes;
+		const { autoplay, caption, controls, loop, muted, preload, src } = attributes;
 		return (
 			<figure>
 				{ src && (
 					<video
 						autoPlay={ autoplay }
 						controls={ controls }
-						src={ src }
 						loop={ loop }
 						muted={ muted }
+						preload={ preload !== 'metadata' ? preload : null }
```

Comment:
I think we should also apply the property to video element used on edit. So the user may be able to notice some difference (if he had not played the video before).

Reply:
If the user adds a video, it preloads by default. When the user changes the preload value, nothing changes because the video is already loaded. If the User selects `none` and reloads the page, there is a visible difference.
I think we can add `muted` and `controls` in a different PR to edit, but `preload` in edit is not very useful for the user because there is no visible change.

Reply:
I agree with @Soean, I don't think we should add the preload to the edit output.

Reply:
Also, we don't use `preload` in [edit on the audio block](https://github.com/WordPress/gutenberg/blob/master/core-blocks/audio/edit.js), so best to be consistent and not use it here.